homeassistant/components/contec_controllers/light.py

```diff
@@ -0,0 +1,73 @@
+"""Contec light entity."""
+
+import logging
+from typing import List
+
+from ContecControllers.ContecOnOffActivation import ContecOnOffActivation
+from ContecControllers.ControllerManager import ControllerManager
+
+from homeassistant.components.light import LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+):
+    """Set up the Contec Lights."""
+    controllerManager: ControllerManager = hass.data[DOMAIN][config_entry.entry_id]
+    allLights: List[ContecLight] = [
+        ContecLight(onOff) for onOff in controllerManager.OnOffActivations
+    ]
+    async_add_entities(allLights)
+
+
+class ContecLight(LightEntity):
+    """Representation of a Contec light."""
+
+    _name: str
+    _onOffActivation: ContecOnOffActivation
+
+    def __init__(self, onOffActivation: ContecOnOffActivation):
+        """Initialize an ContecLight."""
+        self._onOffActivation = onOffActivation
```

Comment:
```suggestion
        self._on_off_activation = onOffActivation
```

Please use snake case for the core code.